README.md

```diff
@@ -26,7 +26,10 @@ Please prefix your file with 3 zero's and a dash (`000-`), it will be renamed to
 
 ## How to test GitHub Pages locally
 
-The [`starefossen/github-pages`](https://hub.docker.com/r/starefossen/github-pages) is a Alpine Docker image for running GitHub Pages / Jekyll projects locally while developing of only 70 MB.
+To build Jekyll, you could [install Jekyll locally](https://jekyllrb.com/docs/installation/) or probably the easier route: use a Docker image.
+To use a Docker image you should have Docker engine installed, see [how to install Docker enginge](https://docs.docker.com/engine/install/).
+
+[`starefossen/github-pages`](https://hub.docker.com/r/starefossen/github-pages) is a Alpine Docker image for running GitHub Pages / Jekyll projects locally while developing of only 70 MB.
```

Comment:
Ace, thanks for the change and addition of link to docs for normal local installation–looks great 👌 

Very minor, but this may read slightly better:

```suggestion
[`starefossen/github-pages`](https://hub.docker.com/r/starefossen/github-pages) is a small Alpine Docker image for running GitHub Pages / Jekyll projects locally.
```